### PR TITLE
Rebase pull req 

### PR DESCRIPTION
**Pull request by awsjoindevops**
- rebase 
- dev is completed

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -4,4 +4,10 @@ This is the frist commit from main branch
 ## Feature-1
 * Developer is doing some development
 
-* Developer spends some time for development
\ No newline at end of file
+* Developer spends some time for development
+
+
+## REbase
+* rebase is another merge strategy
```
Comment (awsdevopsonline) [inline README.MD:11]: is this line required

**Review by awsdevopsonline — COMMENTED** [review]
test

File: README.MD (hunk 1 above)
Comment (awsjoindevops) [reply README.MD:11]: yes it is required

**Review by awsjoindevops — COMMENTED** [review]
yes it is required

**Review by awsdevopsonline — APPROVED** [review]
All Good Approving PR

**Pull request merged**